std/algorithm/searching.d

```diff
@@ -2746,6 +2747,53 @@ if (isForwardRange!R1 && isForwardRange!R2
     assert(findSkip(s, "def") && s.empty);
 }
 
+/**
+* Advances the `haystack` as long as `pred` evaluates to `true`.
+* The `haystack` is positioned before position of first falsely evaluation of `pred`.
```

Comment:
The `haystack` is positioned so as `pred` evaluates to false for `haystack.front`.